### DISCUSSION TITLE
NetTopologySuite library in xamarin forms

### DISCUSSION THREAD

**Discussion by cpalominoch** [NetTopologySuite/NetTopologySuite#494] [Q&A]
Dear,
Can the NetTopologySuite library be used in xamarin forms to create shapefile?

Best regards.

Carlos Palomino

**Comment by airbreather** [comment]
The latest version of NetTopologySuite targets .NET Standard 2.0.  At the time of writing, according to Microsoft, that means it can run on the following platforms (as of the listed version or later):

-    .NET Core 2.0
-    .NET Framework 4.6.1 (but there are massive problems that weren't fully fixed until 4.7.2)
-    Mono 5.4
-    Xamarin.iOS 10.14
-    Xamarin.Mac 3.8
-    Xamarin.Android 8.0
-    Universal Windows Platform 10.0.16299
